src/TLuaInterpreter.cpp

```diff
@@ -11665,6 +11687,12 @@ void TLuaInterpreter::adjustCaptureGroups(int x, int a)
             it += a;
         }
     }
+
+    QMap<QString, QPair<int,int>>::iterator i;
+    for (i = mCapturedNameGroupsPosList.begin(); i != mCapturedNameGroupsPosList.end(); ++i) {
```

Comment:
Since we don't do any modification - 

```suggestion
    for (i = mCapturedNameGroupsPosList.cbegin(); i != mCapturedNameGroupsPosList.cend(); ++i) {
```

Reply:
This change would create compilation error, so left as it is.